docs/README.md

```diff
@@ -0,0 +1,250 @@
+# teracy-dev docs
+
+Follow this guide to work on the docs
+
+## Prerequisites
+
+- `docker-registry` must be available by following https://github.com/teracyhq-incubator/teracy-dev-k8s/blob/develop/docs/docker-registry.md
+
+- `docker`, `kubectl`, `helm`, `skaffold` must be installed on your host machine and make sure it works
+  with the k8s cluster
+
+## Set default-repo for Skaffold
+
+```bash
+$ skaffold config set default-repo registry.k8s.local
+```
+
+## Skaffold dev mode with kubectl
+
+- Execute the following commands:
+
+```bash
+$ cd docs
+$ skaffold dev
```

Comment:
Not work with `Docker for Desktop` and our k8s vm on Windows with the same reason https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/82/files#r264996507

Reply:
we're going to use `docker daemon remote access` so that it should work on Windows then.